apps/web/src/components/reporting/ReportTable.tsx

```diff
@@ -44,7 +44,8 @@ export const ReportTable = () => {
   };
 
   const download = async (period: Period) => {
-    const from = order === 'ASC' ? periods[0].from : periods[periods.length - 1].from;
```

Comment:
This should be reverted cause line 83 is enough.

Reply:
this just causes all the start dates to be Apr 01, 2021